src/main/scala/atom/Atom.scala

```diff
@@ -46,7 +46,7 @@ class CommandListener(
   val displayName: js.UndefOr[String] = js.undefined,
   val description: js.UndefOr[String] = js.undefined,
   val hiddenInCommandPalette: js.UndefOr[Boolean] = js.undefined
-)(callback: js.Any => Unit) extends js.Object {
+)(callback: js.Function1[js.Any, Unit]) extends js.Object {
```

Comment:
This was done so with the intention to simplify usage, but now it's pointless. So you could simplify this a bit:

```
)(val didDispatch: js.Function1[js.Any, Unit]) extends js.Object
```

(no need in the intermediate `callback` parameter)